src/components/breadcrumb/breadcrumb.js

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Breadcrumb } from 'gatsby-plugin-breadcrumb';
 
-const Breadcrumbs = ({ crumbs, crumbLabel, locationOrigin }) => {
+const mainSiteUrl = process.env.SITE_URL || 'https://ssw.com.au';
```

Comment:
this seems to be duplicated with the header - would prefer this to be encapsulated in a site config object

Reply:
header and breadcrumbs are loaded in the Layout.js. So, the new site-config variable is imported there and passed to both places.